deb/verify

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+###
+# Script Name:  verify
+#
+# Description: This runs a smoke test to verify that the packages can be installed correctly
+###
+
+set -e -x extglob
+
+# Steps taken from: https://docs.docker.com/install/linux/docker-ce/ubuntu/
+function install_containerd() {
+    echo "[DEBUG] Installing containerd"
+    curl -fsSL https://download.docker.com/linux/${DIST_ID}/gpg | apt-key add -
+
+    if [[ ${DIST_VERSION}  == "sid" ]]; then
+        DIST_VERSION='buster'
+    fi
+    echo "deb [arch=${ARCH}] https://download.docker.com/linux/${DIST_ID} \
+        ${DIST_VERSION} \
+        stable" > /etc/apt/sources.list.d/docker.list
+
+    apt-get update
+}
+
+
+function verify() {
+    apt-get update
+    apt-get -y install lsb-release software-properties-common curl apt-transport-https gnupg2
+
+    DIST_ID=$(source /etc/os-release; echo "$ID")
+    DIST_VERSION=$(lsb_release -sc)
+    ARCH=$(dpkg --print-architecture)
+
+    install_containerd
+
+    packages=$(find "/build" -type f -name "*.deb")
+    # All local packages need to be prefixed with `./` or else apt-get doesn't understand where to pull from
+    packages=$(echo "${packages}" | awk '$0="."$0' | xargs)
+
+    (
+        set -x
+        # Install the locally built packages using 'dpkg' because installing with
+        # 'apt-get' would attempt to install dependency packages (such as the CLI)
+        # from download.docker.com instead of the locally built CLI package. Given
+        # that 'dpkg -i' does not install any dependency (but will fail if depen-
+        # dencies are missing), we use the '--ignore-depends' option to ignore
+        # packages we know to be missing at this stage, and '--force-depends' to
+        # only warn about any other missing dependency.
+        #
+        # Afterwards, we run 'apt-get install' with the '--fix-broken' option to
+        # trigger installation of the dependencies, which should succceed succesfully.
+        dpkg \
```

Comment:
You can also achieve this with just a simple `apt-get install ./${packages}`, that's worked to install local packges for a while now while still using the same `apt-get` resolver

Reply:
ISTR there was _something_ that didn't work, but don't recall what. Does it also support the `--ignore-depends` and `--force-depends` options?

Reply:
It should be compatible with all flags, idk why apt doesn't just work with local files without adding the prefixed `./` but it should *just work*.

This is the verification model I was using for `containerd.io` before leaving.

Reply:
Thanks @seemethere ! I'll try that in a follow-up as this PR is mainly moving code 👍